src/goto-programs/lazy_goto_model.cpp

```diff
@@ -207,6 +208,24 @@ void lazy_goto_modelt::load_all_functions() const
 
 bool lazy_goto_modelt::freeze()
 {
+  messaget msg(language_files.get_message_handler());
+  journalling_symbol_tablet symbol_table=
+    journalling_symbol_tablet::wrap(this->symbol_table);
+  if(language_files.final(symbol_table))
+  {
+    msg.error() << "FINAL STAGE CONVERSION ERROR" << messaget::eom;
```

Comment:
What would a user know about "FINAL STAGE"? (Actually even as a developer I don't know what that is.)

Reply:
Agreed. This has been copied from initialize_goto_model. Someone suitably qualified to know what the message should be should fix it there and I'll copy the fix in here if that merges first.

Reply:
That was 98643d7335 - @kroening ?!

Reply:
I don't think the user knows about 'FINAL STAGE'. 'CONVERSION ERROR' alone is sufficient (even though I can imagine better error messages than 'CONVERSION ERROR' -- we can address this when fixing up the error handling. Then an appropriate error message can be produced at the location where the 'ERROR' happens.